torch/_inductor/codegen/cpp.py

```diff
@@ -1347,7 +1349,9 @@ class LoopLevel:
     steps: sympy.Expr = sympy.Integer(1)
     parallel: int = 0
     simd_omp: bool = False
-    simd_len: int = config.cpp.simdlen
+    simd_len: int = (
+        codecache.pick_vec_isa().nelements() if codecache.pick_vec_isa() else 0
+    )
```

Comment:
maybe keep the `VecISA` object directly here would be simpler?

Reply:
Done